src/Python/sirf/contrib/NEMA/generate_nema_rois.py

```diff
@@ -0,0 +1,257 @@
+'''Generate NEMA ROIs.
+
+Usage:
+  generate_nema_rois [--help | options]
+
+Options:
+  -s <file>, --sino=<file>     raw data file [default: no default you need an input of a NEMA sinogram]
```

Comment:
```suggestion
  -s <file>, --sino=<file>     raw data file (no default value as you need an input of a NEMA sinogram)
```
to avoid problems with `docopt`